src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,51 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+import mate.academy.model.impl.BookImpl;
+
 public class Main {
+    private static final Injector injector = Injector
+            .getInstance("mate.academy.dao.impl");
+
     public static void main(String[] args) {
+        Book book = new BookImpl();
+        book.setId(1L);
```

Comment:
Id must be generated by the database. Do not hardcode it in the code
don't hardcode ids, retrieve by getId()
```suggestion
```